webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -16,25 +18,44 @@ export const getResults = ({
   searchQuery,
   controller,
   trigger,
+  id,
 }) => dispatch => {
   startRequest({
     controller,
     searchQuery,
     trigger,
     dispatch,
+    url,
+    id,
   });
-
+  if (!url) {
+    return requestFailure({
+      error: new Error(__('No API path was provided.')),
```

Comment:
This is very confusing to users. 
This isn't an error they have anything to do about - the error is in the code; displaying an error to users should only be done when they might have something they can do or we have no other choice. In this case, it would make sense to display an error if the api request failed, or if they tried to input a field that isn't searchable.
In this case, i think the autocomplete should just turn into a regular text input so they can input a search even if we don't autocomplete (for example, auth sources do support some searching but don't have a controller that provides autocomplete endpoint)

Reply:
Thanks,
whether this is a filter-page error that wasn't detected before,
and when this bugs will be fixed there won't be any error?
or is it a UX thing that we shouldn't show this kind of errors..

I can agree on both, but I also think that this should be discussed in another PR,
with the help of UX designers and maybe the community,
to discuss which errors we want to show and which we doesn't,
and how to face them.

maybe just showing a console error would be good,
but then, if I have autocomplete and it doesn't work and the user don't see an error,
is it an autocomplete bug? I guess most of the users thought so when the autocomplete didn't work on some of the resources on this page.

Reply:
This error didn't display previously, the field worked, it just didn't autocomplete - so you could fill it in if you knew what you wanted to search for. So i don't think this should be split to a separate pr - as users will now see a confusing error message on a field they used to be able to enter an input on.

Reply:
> the field worked, it just didn't autocomplete

which means it didn`t work :) should we leave it as non-functioning input than?
Lets open another PR for this, or fix the root of the errors by https://projects.theforeman.org/issues/25652

Reply:
the autocomplete didn't work, but the input itself works just fine. We shouldn't display an error message to the user in this case.

Reply:
fixed, now the `no-url-error` would be shown only in the console, without indication to the user.